controllers/delete.go

```diff
@@ -34,7 +34,7 @@ func (r *StorageSystemReconciler) deleteResources(instance *odfv1alpha1.StorageS
 
 	var backendStorage client.Object
 
-	r.deleteQuickStarts(logger)
+	r.deleteQuickStarts(logger,instance)
```

Comment:
here as well

Reply:
Checked & updated all the occurances. Now should be good to go.